.github/workflows/push-images.yml

```
@@ -0,0 +1,48 @@
+name: Push-Images
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+          
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          
+      - ame: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Gather Docker Meta Data
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ghcr.io/${{ github.actor }}/mlflow
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=ref,event=tag
+      
+      - name: Build and Push Images
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
```

```suggestion
          push: true
          file: <path to Dockerfile>
```

I think we need to specify `file`.